lib/foreman_remote_execution/engine.rb

```diff
@@ -61,6 +57,100 @@ class Engine < ::Rails::Engine
+                              'This setting only applies to IPv4. When the host has only an IPv6 address on the interface used for remote execution, hostname will be used even if this setting is set to true.'),
+              default: false,
+              full_name: N_('Connect by IP')
+            setting 'remote_execution_ssh_password',
+              type: :string,
+              description: N_('Default password to use for SSH. You may override per host by setting a parameter called remote_execution_ssh_password'),
+              default: nil,
+              full_name: N_('Default SSH password'),
+              encrypted: true
+            setting 'remote_execution_ssh_key_passphrase',
+              type: :string,
+              description: N_('Default key passphrase to use for SSH. You may override per host by setting a parameter called remote_execution_ssh_key_passphrase'),
+              default: nil,
+              full_name: N_('Default SSH key passphrase'),
+              encrypted: true
+            setting 'remote_execution_workers_pool_size',
+              type: :integer,
+              description: N_('Amount of workers in the pool to handle the execution of the remote execution jobs. Restart of the dynflowd/foreman-tasks service is required.'),
+              default: 5,
+              full_name: N_('Workers pool size')
+            setting 'remote_execution_cleanup_working_dirs',
+              type: :boolean,
+              description: N_('When enabled, working directories will be removed after task completion. You may override this per host by setting a parameter called remote_execution_cleanup_working_dirs.'),
+              default: true,
+              full_name: N_('Cleanup working directories')
+            setting 'remote_execution_cockpit_url',
+              type: :string,
+              description: N_('Where to find the Cockpit instance for the Web Console button.  By default, no button is shown.'),
+              default: nil,
+              full_name: N_('Cockpit URL')
+            setting 'remote_execution_form_job_template',
+              type: :string,
+              description: N_('Choose a job template that is pre-selected in job invocation form'),
+              default: 'Run Command - SSH Default',
+              full_name: N_('Form Job Template'),
+              collection: proc { Hash[JobTemplate.unscoped.map { |template| [template.name, template.name] }] }
+            setting 'remote_execution_job_invocation_report_template',
+              type: :string,
+              description: N_('Select a report template used for generating a report for a particular remote execution job'),
+              default: 'Jobs - Invocation report template',
+              full_name: N_('Job Invocation Report Template'),
+              collection: proc { self.job_invocation_report_templates_select }
```

Comment:
If I'm looking right then `self.job_invocation_report_templates_select` no longer exists.

Reply:
:+1: but also I think that it should be removed from this file: https://github.com/theforeman/foreman_remote_execution/blob/e492aef4b0de384011990b7ddb28447dd3d3c8bf/test/unit/job_invocation_report_template_test.rb